sale_order_line_sequence/views/report_saleorder.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template
+        id="report_saleorder_document_sequence"
+        inherit_id="sale.report_saleorder_document"
+    >
+        <xpath expr="//table[hasclass('table')]/thead/tr/th[1]" position="before">
```

Comment:
Can you make the table selection more specific? I see multiple tables with this class in the document. I think each does have a unique class that you can add here.